src/components/select/select.ios.scss

```diff
@@ -12,6 +12,11 @@ $select-ios-padding-left:        $item-ios-padding-left !default;
 $select-ios-icon-color:          #999 !default;
 $select-ios-placeholder-color:   $select-ios-icon-color !default;
 
+$select-popover-ios-margin-top:         -1px !default;
+$select-popover-ios-margin-right:       0 !default;
+$select-popover-ios-margin-bottom:      -1px !default;
+$select-popover-ios-margin-left:        0 !default;
```

Comment:
Could these be changed to start with `$select-ios-popover-`? I like to use the following order for naming variables:

```
<component>-<mode>-<child element or component>-<css property>
```

See the following document for more info: https://docs.google.com/document/d/1OyOyrRE5lpB_9mdkF0HWVQLV97fHma450N8XqE4mjZQ/edit

Thanks :)